src/main/kotlin/io/curity/identityserver/plugin/frejaeid/authentication/StartRequestHandler.kt

```diff
@@ -142,45 +146,72 @@ class StartRequestHandler(private val config: FrejaEidAuthenticatorPluginConfig,
 
         if (statusCode != 200)
         {
-            if (_logger.isErrorEnabled)
+            if (_logger.isWarnEnabled)
             {
                 _logger.warn("Got error response from authentication endpoint: error = {}, {}", statusCode,
                         httpResponse.body(HttpResponse.asString()))
             }
 
+            if (_json.fromJson(httpResponse.body(HttpResponse.asString()))["code"] == 2000)
```

Comment:
2000 is right? It's not a typo and should be 200?

Reply:
Yes, 2000 is correct, as described here: https://frejaeid.com/rest-api/Authentication%20Services.html